### ISSUE TITLE
Update Links in README.md before making a release

### ISSUE THREAD

**Issue by msetina** [OpenSC/OpenSC#3035]
README file contains:
Do NOT use any links from wiki to download the OpenSC because wiki can be modified by anybody, see
[#2554](https://github.com/OpenSC/OpenSC/issues/2554). For downloading OpenSC, use the links here in README.

# Downloads

[OpenSC 0.23.0](https://github.com/OpenSC/OpenSC/releases/tag/0.23.0) is the latest stable version released on
29.11.2022. It is available as

 * Windows installer
   * [OpenSC-0.23.0_win64.msi](https://github.com/OpenSC/OpenSC/releases/download/0.23.0/OpenSC-0.23.0_win64.msi) for 64 bit programs
   * [OpenSC-0.23.0_win32.msi](https://github.com/OpenSC/OpenSC/releases/download/0.23.0/OpenSC-0.23.0_win32.msi) for 32 bit programs
 * [OpenSC-0.23.0.dmg](https://github.com/OpenSC/OpenSC/releases/download/0.23.0/OpenSC-0.23.0.dmg): macOS installer
 * [opensc-0.23.0.tar.gz](https://github.com/OpenSC/OpenSC/releases/download/0.23.0/opensc-0.23.0.tar.gz): Source code distribution

**Comment by frankmorgner** [comment]
We will not change the 0.24.0 artifacts anymore, but with your comment in mind, we'll try to make sure that 0.25.0 files contain the correct links

**Comment by Jakuje** [comment]
Thank you for the suggestion! I did it after the release last time as files exist only after we tag the release, the artifacts are built and uploaded to the release page itself. The paths are fixed so it can be certainly done before the release. But I wanted to avoid to have a dead links on the main page of the project before all the stuff will be ready.
Other option might be going back to the wiki given that we did not open it for modification of non-members (and I do not think we will do it any time soon).
